src/backend/distributed/sql/udfs/worker_repartition_cleanup/9.1-1.sql

```diff
@@ -0,0 +1,8 @@
+CREATE FUNCTION pg_catalog.worker_repartition_cleanup(bigint)
```

Comment:
maybe worker_job_cleanup

Reply:
I thought that as what we are doing is specific to repartition, it is better to be explicitly state it here. Like we cleanup repartition specific things. What do you think?

Reply:
Actually, could we reuse task_tracker_cleanup_job here? What's the difference between those functions?

Reply:
`task_tracker_cleanup_job ` has task tracker specific hash table and it removes tasks from the hash table which was filled by task tracker but later it removes the directory and schema as well. So they have common parts so I think we can use the `worker_repartition_cleanup` to do remove directory and schema in task tracker as well, but using `task_tracker_cleanup_job ` seems wrong to me because it is task tracker specific.